bundles/SyliusReviewBundle/container-configuration.rst

```diff
@@ -0,0 +1,39 @@
+Container configuration
+=======================
+
+Using *SyliusReviewBundle*, you should configure your review resource with several key. They'll be automatically processed
+to proper configuration and database mapping. Whole configuration should be written under ``sylius_review`` tag in your ``app/config/config.yml`` file.
+Under ``sylius_review``, ``classes`` key is required, which defines tree of specific reviews configuration. Next tag should be named according
```

Comment:
, which defines tree of specific reviews configuration. => and defines configuration tree for specific review ?